physics/resonance_test.cpp

```diff
@@ -0,0 +1,391 @@
+﻿#include <map>
+#include <vector>
+
+#include "physics/kepler_orbit.hpp"
+#include "rigid_motion.hpp"
+#include "gmock/gmock.h"
+#include "gtest/gtest.h"
+#include "mathematica/mathematica.hpp"
+#include "physics/solar_system.hpp"
+#include "quantities/astronomy.hpp"
+#include "testing_utilities/almost_equals.hpp"
+#include "testing_utilities/numerics.hpp"
+
+namespace principia {
+
+using integrators::McLachlanAtela1992Order5Optimal;
+using quantities::astronomy::JulianYear;
+using quantities::si::Degree;
+using quantities::si::Kilo;
+using quantities::si::Metre;
+using quantities::si::Milli;
+using testing_utilities::RelativeError;
+
+namespace physics {
+
+#if !defined(_DEBUG)
```

Comment:
I would only ifdef the expensive test.  All the rest might as well stay to help us debug regressions.